packages/playground/src/utils/itemResizer.js

```diff
@@ -79,6 +79,8 @@ const resizeUrlImp = (
       options.filters = {
         blur: sharpParams.blur,
       };
+      //this is a hack to avoid using the imageClientSdk for blurry images. These have to stay EXACTLY the same between SSR and CSR
+      return `${item.mediaUrl}/v1/fit/w_250,h_250,al_c,q_30,blur_30/${item.id}.jpg`;
```

Comment:
This will force this url on any blur. Why not create a function on the galleryItem for this where the rest are.

createUrl('largeThumbnail','blur')  something like that... I dont remember the actual enum naming and options.

Then you use this in places where you want to get this type of constant representation of a mediaUrl.

Reply:
Yes, we should do that...
I don't want to add it here though